spec/libraries/azure_engine_spec.rb

```diff
@@ -25,15 +26,18 @@ def delete_message(queue_name, message_object_id, message_object_pop_receipt)
       end
     end
   end
+
   module Entity
     module Queue
       class DummyMessage
-        def id
-          1
-        end
-
-        def pop_receipt
-          ''
+        attr_accessor :id
+        attr_accessor :message_text
+        attr_accessor :pop_receipt
+
+        def initialize
```

Comment:
__[method_definition_validator]__
* Lint/DefinitionValidator: pop_receipt is undefined. Did you mean? initialize at [lib/slide_hub/cloud/engine/azure.rb#L60](/ryuzee/SlideHub/blob/9b8071729d12c9d8fedb90368fc91a2d3bf9c1f5/lib/slide_hub/cloud/engine/azure.rb#L60)

<!-- Comment by [SideCI](https://sideci.com) -->